app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -32,14 +32,16 @@ EOF
 
 
 register_host() {
-  curl --cacert $SSL_CA_CERT --request POST <%= foreman_server_url %>/register \
+  curl --cacert $SSL_CA_CERT --request POST '<%= foreman_server_url %>/register' \
        <%= headers.join(' ') %> \
-       -d '{ "host": { "name": "'$(hostname --fqdn)'", "build": "false", "managed": "false"
-       <%= ", \"organization_id\": \"#{@organization.id}\"" if @organization -%>
-       <%= ", \"location_id\": \"#{@location.id}\"" if @location -%>
-       <%= ", \"hostgroup_id\": \"#{@hostgroup.id}\"" if @hostgroup -%>
-       <%= ", \"operatingsystem_id\": \"#{@operatingsystem.id}\"" if @operatingsystem -%>
-       }}'
+       --data "host[name]=$(hostname --fqdn)" \
+       --data 'host[build]=false' \
+       --data 'host[managed]=false' \
+       <%= "--data 'host[organization_id]=#{@organization.id}'" if @organization -%>
+       <%= "--data 'host[location_id]=#{@location.id}'" if @location -%>
+       <%= "--data 'host[hostgroup_id]=#{@hostgroup.id}'" if @hostgroup -%>
+       <%= "--data 'host[operatingsystem_id]=#{@operatingsystem.id}'" if @operatingsystem -%>
```

Comment:
Shouldn't all the strings end with `\` to mark that the command continues on next line?

Reply:
Yes, that looks like a bug in #8068.

Reply:
yes, fixing here to make it work

Reply:
Oh, it is not a bug (and never was), because it ends on the same line (`-%>`), it's not formatted like the lines above. However, I think it will be nicer if we format it the same way. So I'll add `\` and remove the `-` from closing erb tags.